app/celery/onsite_notification_tasks.py

```diff
@@ -6,9 +6,9 @@
 @notify_celery.task(name="send-va-onsite-notification-task")
 def send_va_onsite_notification_task(va_profile_id: str, template_id: str, onsite_enabled: bool = False):
     """ This function is used by celery to POST a notification to VA_Onsite. """
-    current_app.logger.info(f'Calling va_onsite_notification_task with va_profile_id: {va_profile_id}\n'
-                            f'template_id: {template_id}\n'
-                            f'onsite_notification set to: {onsite_enabled}')
+    current_app.logger.info(
+        "Calling va_onsite_notification_task with va_profile_id: %s\ntemplate_id: %s\nonsite_notification set to: %s",
```

Comment:
do we really need the newlines `\n` here? is there a specific reason why this can't be logged on a single line?

Reply:
I think I originally wrote this and just thought it was easier to see that way. I don't think there's any specific reason for it to be that way.